casbin_pymongo_adapter/adapter.py

```diff
@@ -1,35 +1,98 @@
+import casbin
 from casbin import persist
 from pymongo import MongoClient
 
+class CasbinRule():
+    '''
+    CasbinRule model
+    '''
+
+    __tablename__ = "casbin_rule"
+
+    def __init__(self, ptype='', v0='', v1='', v2='', v3='', v4='', v5=''):
+        self.ptype = ptype
+        self.v0 = v0
+        self.v1 = v1
+        self.v2 = v2
+        self.v3 = v3
+        self.v4 = v4
+        self.v5 = v5
+
+    def __str__(self):
+        dict = {'ptype':self.ptype}
+        if self.v0:
+            dict['v0'] = self.v0
+        if self.v1:
+            dict['v1'] = self.v1
+        if self.v2:
+            dict['v2'] = self.v2
+        if self.v3:
+            dict['v3'] = self.v3
+        if self.v4:
+            dict['v4'] = self.v4
+        if self.v5:
+            dict['v5'] = self.v5
+        return dict
+
+    def __repr__(self):
+        return '<CasbinRule :"{}">'.format(str(self))
+

```

Comment:
Since `pymongo` uses `dictionary` types as mongodb `documents`, CasbinRule should implement the method of conversion to `dictionary`。

```python
class CasbinRule:
    ptype = None
    v0 = None
    v1 = None
    v2 = None
    v3 = None
    v4 = None
    v5 = None

    def __init__(self, ptype):
        self.ptype = ptype

    def dict(self):
        d = {'ptype': self.ptype}

        for i, v in enumerate([self.v0, self.v1, self.v2, self.v3, self.v4, self.v5]):
            if v is None:
                break
            d['v' + str(i)] = v

        return d

    def __str__(self):
        return ', '.join(self.dict().values())

    def __repr__(self):
        return '<CasbinRule :"{}">'.format(str(self))
```

Reply:
Oh, I see that. I tried to fix the dictionary type before, but there may be sth wrong with my implementation.